contrib/ruby_event_store-outbox/spec/tempo_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "spec_helper"
+
+module RubyEventStore
+  module Outbox
+    RSpec.describe Tempo do
+      specify "starts with 1" do
+        tempo = Tempo.new(100)
+
+        expect(tempo.batch_size).to eq(1)
+      end
+
+      specify "each batch size is bigger" do
```

Comment:
`each batch size is bigger` but testing only 2 first values ;)

Reply:
You're right, "many", is three :joy: I will change that